app/gui/src/presenter/graph/state.rs

```diff
@@ -532,7 +532,7 @@ impl<'a> ControllerChange<'a> {
             None | Some(Value) => None,
             Some(DataflowError { trace }) => Some((Kind::Dataflow, None, trace)),
             Some(Panic { message, trace }) => Some((Kind::Panic, Some(message), trace)),
-            Some(Pending { .. }) => None,
+            Some(Pending { .. }) => Some((Kind::Pending, Some("Pending...".to_string()), Vec::new())),
```

Comment:
extract literals to consts, please.

Reply:
Resolved in [bca0fbb](https://github.com/enso-org/enso/pull/4088/commits/bca0fbb8162d35cc201dc0dd9ed0bc976dea3e0a)